data-collection/src/main/java/refactoringml/ProcessMetricsCollector.java

```diff
@@ -392,7 +395,7 @@ private void outputNonRefactoredClass (ProcessMetric clazz) throws IOException {
 					commitHashBackThen,
 					commitDate,
 					ck.getFile().replace(tempDir, ""),
-					ck.getClassName(),
+					cleanedCkClassName,
```

Comment:
Good catch! I think it's fine to use the clean name now that we have a boolean indicating whether this is a subclass!